web/pages/bus.vue

```diff
@@ -87,14 +87,10 @@ export default {
       // static build -> no mensa plan is in the store
       this.load()
     }
-    if (process.client) {
-      this.refreshTimer = setInterval(() => this.load(), 60000)
-    }
+    this.refreshTimer = setInterval(() => this.load(), 60000)
```

Comment:
Das hier hab ich entfernt, da es laut der neuen ESLint-Regel nicht notwendig ist. Da müsstest ihr nochmal rüberschauen, da ich durch NUXT neu nicht so ganz durchgestiegen bin. Vom Testen her funktioniert aber noch alles so wie vorher.

https://github.com/nuxt/eslint-plugin-nuxt/blob/master/docs/rules/no-env-in-hooks.md

Reply:
Stimmt, der Check war überflüssig.